index.js

```diff
@@ -76,19 +78,26 @@ if (!output && !dir && !argv.replace) {
 // Use warn to avoid writing to stdout
 console.warn(chalk.bold.red(logo))
 
-spinner.text = `Loading Config`
+spinner.text = 'Loading Config'
 spinner.start()
-Promise.all([ globber(input), getConfig({}, argv.config) ])
-  .then((arr) => {
-    // Until parameter destructuring is supported
-    let files = arr[0]
+getConfig({}, argv.config)
+  .then(() => {
+    spinner.succeed()
 
-    if (!files || !files.length) throw new Error('You must pass a list of files to parse')
+    if (input && input.length) return globber(input)
+    // Read from stdin:
+    else {
+      if (!argv.output) throw new Error('Must specify --output when reading from stdin')
```

Comment:
This is mainly to throw an error if `-d` or `-r` is passed. I'm thinking about enabling stdout support, just haven't done yet since v2 didn't support stdout (v2 feature parity is first priority).